### PR TITLE
Restoring `-standalone` for PoS

### PR DESCRIPTION
**Pull request by troels-im**
With this PR, `-standalone` works again using PoS and enables specifying which account all funds initially belongs to. Example:

```
./apps/constellation/constellation -standalone -network local -initial-address 2KP7BjerxYJTtKtAsyUi2xzDyVTtz8Ynn7E2ANxjBqNv4oHZ3j
```

### CODE REVIEW

File: apps/constellation/config_builder.cpp [hunk 1]
```diff
@@ -314,7 +315,33 @@ Constellation::Config BuildConstellationConfig(Settings const &settings)
...
+
+    variant::Variant config = variant::Variant::Object();
+    config["version"]       = 4;
+    config["accounts"]      = accounts;
+
+    std::stringstream contents{""};
+    contents << config;
+
+    cfg.genesis_file_contents = contents.str();
+    cfg.block_interval_ms     = 3000;
```
Comment (ejfitzgerald) [inline apps/constellation/config_builder.cpp]:
```suggestion
```

File: apps/constellation/config_builder.cpp [hunk 2]
```diff
@@ -314,7 +315,33 @@ Constellation::Config BuildConstellationConfig(Settings const &settings)
...
+    cfg.aeon_period           = 0;
```
Comment (ejfitzgerald) [inline apps/constellation/config_builder.cpp]:
```suggestion
```

File: apps/constellation/config_builder.cpp [hunk 3]
```diff
@@ -314,7 +315,33 @@ Constellation::Config BuildConstellationConfig(Settings const &settings)
...
+    cfg.proof_of_stake        = false;
```
Comment (ejfitzgerald) [inline apps/constellation/config_builder.cpp]:
```suggestion
```

File: apps/constellation/config_builder.hpp [hunk 4]
```diff
@@ -18,6 +18,7 @@
 //------------------------------------------------------------------------------
 
 #include "constellation/constellation.hpp"
+#include "crypto/identity.hpp"
```
Comment (ejfitzgerald) [inline apps/constellation/config_builder.hpp]: Why needed?

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]

File: apps/constellation/config_builder.hpp (hunk 4 above)
Comment (troels-im) [inline apps/constellation/config_builder.hpp]:
```suggestion
```

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**